examples/MultipleOctofet/MultipleOctofet.ino

```diff
@@ -0,0 +1,43 @@
+// Two libraries have to be included
+// to work with the module. Download at
+// https://amperka.com/my/octofet
+#include <Octofet.h>
+#include <SPI.h>
+
+// any GPIO pin can be used as pinCS
+constexpr auto pinCS = 10;
```

Comment:
Igor, is it worth it to use **constexpr** in the examples calculated for a simple arduino user, and even **auto** to the same. We will confuse them. This question is not so much for you as for Victor.

Reply:
We demonstrate good practice here. It is better than the traditional:

```cpp
#define PIN_CS 10
```

Let it be.

Reply:
Ok, leaved as is.